### PR TITLE
Fix the flush in gen.py (not a submission)

### PR DESCRIPTION
**Pull request by webNeat**
Hello,

I fixed a small bug in the generator script to have the expected flush behaviour.

### CODE REVIEW

File: gen.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def main():
         for cur in range(args.rows):
             line = random.choice(moroccan_cities) + "," + random.choice(fruits_and_vegetables) + "," + str(generate_random_decimal())
             lines += line + '\n'
-            if cur % args.flush:
+            if not cur % args.flush:
```
Comment (chermehdi) [inline gen.py:83]: nit: can you make it more explicit
```suggestion
            if cur % args.flush == 0:
```

**Review by chermehdi — CHANGES_REQUESTED** [review]
Thanks for fixing the bug

File: gen.py (hunk 1 above)
Comment (webNeat) [reply gen.py]: Done

**Review by aboullaite — APPROVED** [review]

**Comment by YassineOsip** [comment]
@chermehdi I think you forgot to remove csv import statement :)

**Pull request merged**